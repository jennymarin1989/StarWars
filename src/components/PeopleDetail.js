import React from 'react';


const PeopleDetail = () => {
  return (
    <div>
     <h1>StarWars Character details:</h1> 
    </div>  
  )
}

export default PeopleDetail;