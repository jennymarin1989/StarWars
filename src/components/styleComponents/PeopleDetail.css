.detail {
  font-family: verdana;
  font-weight: bold;
  font-size: 1.1em;
  color: #FFD700
}