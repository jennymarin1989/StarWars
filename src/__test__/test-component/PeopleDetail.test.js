import React from 'react';
import { shallow, mount } from 'enzyme';
import PeopleDetail from '../../components/PeopleDetail';

describe('PeopleDetail component', () => {

  describe('renders components', ()=> {
    it('should render components correctly', () => {
      let detail = shallow(<PeopleDetail />);
      expect(detail).toMatchSnapshot();
    });
  });

  it('should pass props from parent component correctly', () => {
    let peopleDetail = mount(<PeopleDetail details = {"Luke SkyWalker"}/>);
    expect(peopleDetail.props().details).toBe("Luke SkyWalker");
  })

})