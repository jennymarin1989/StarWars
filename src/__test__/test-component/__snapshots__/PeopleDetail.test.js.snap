// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeopleDetail component renders components should render components correctly 1`] = `<PeopleDetail />`;
